backend/MonChenil/Controllers/TimeSlotsController.cs

```diff
@@ -72,4 +72,13 @@ public IActionResult Delete(int id)
         timeSlotService.Delete(timeSlot);
         return Ok();
     }
+
+    [HttpGet("available")]
+    public IActionResult GetAvailableTimeSlots()
+    {
+        var availableTimeSlots = timeSlotService.GetAvailableTimeSlots(
+            new List<Pet>()
+        );
+        return Ok(availableTimeSlots);
+    }
```

Comment:
Tu peux passer prendre un liste d'id de pets en argument ?

Reply:
Did it in my last commit #https://github.com/MonChenil/monchenil/pull/23/commits/0998fd9d611ed4c18275557534b16442494d03a7